src/tests/ftest/datamover/copy_procs.py

```diff
@@ -46,46 +44,47 @@ def setUp(self):
         super(CopyProcsTest, self).setUp()
 
         # Get the parameters
-        self.test_file = self.params.get(
-            "test_file", "/run/ior/*")
-        self.flags_write = self.params.get(
-            "flags_write", "/run/ior/copy_procs/*")
-        self.flags_read = self.params.get(
-            "flags_read", "/run/ior/copy_procs/*")
+        self.ior_flags = self.params.get(
+            "ior_flags", "/run/ior/*")
+        self.flags_write = self.ior_flags[0]
+        self.flags_read = self.ior_flags[1]
```

Comment:
Optional: We can avoid the use of these extra params by directly using self.ior_flags[0] and self.ior_flags[1]